com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -366,13 +365,15 @@ private void Init(bool server)
             SnapshotSystem = new SnapshotSystem();
 
             //This 'if' should never enter
-            if (NetworkTickSystem != null)
+            if (NetworkTimeSystem != null)
             {
-                NetworkTickSystem.Dispose();
-                NetworkTickSystem = null;
+                NetworkTimeSystem.NetworkTickInternal -= OnNetworkTick;
+                NetworkTimeSystem = null;
```

Comment:
I'm assuming the .Dispose() call was extraneous then?  I wonder if @TwoTenPvP ran into some kind of cleanup bug that led him to add this before.

Reply:
@jeffreyrainy was the one who created the NetworkTickSystem. The reason why there is no dispose anymore is because the new time system is no longer an `IDisposable` so we do not need/can call dispose on it.

Reply:
Also I will change this back to use IDisposable again to be able to completely decouple this from NetworkManager.